### ISSUE TITLE
How to copy a convolution filter?

### ISSUE THREAD

**Issue by mike-shvets** [dmlc/mxnet#1319]
I need to build a network, similar to pretrained Inception-BatchNorm Network, but with different sizes of layers (as my images are bigger than 224x224 and resizing is not the way to go).

I want to initialize convolution layers with pretrained filters from that network. What is the way to get the filters from pretrained model and set to a new one?

**Comment by piiswrong** [comment]
Duplicate https://github.com/dmlc/mxnet/issues/1313
